insights/2015-04-01/swagger/insightsClient_TenantEvents.json

```diff
@@ -0,0 +1,297 @@
+          "enum": [
+            "Admin",
+            "Operation",
+            "Debug",
+            "Analytics"
+          ],
+          "x-ms-enum": {
+            "name": "EventChannels",
+            "modelAsString": false
+          }
+        },
+        "claims": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "key value pairs to identify ARM permissions."
+        },
+        "caller": {
+          "type": "string",
+          "description": "the email address of the user who has performed the operation, the UPN claim or SPN claim based on availability."
+        },
+        "description": {
+          "type": "string",
+          "description": "the description of the event."
+        },
+        "id": {
+          "type": "string",
+          "description": "the resource Id."
+        },
+        "eventDataId": {
+          "type": "string",
+          "description": "the event data Id. This is a unique identifier for an event."
+        },
+        "correlationId": {
+          "type": "string",
+          "description": "the correlation Id, usually a GUID in the string format. The correlation Id is shared among the events that belong to the same uber operation."
+        },
+        "eventName": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "the event name. This value should not be confused with OperationName. For practical purposes, OperationName might be more appealing to end users."
+        },
+        "category": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "the event category."
+        },
+        "httpRequest": {
+          "$ref": "#/definitions/HttpRequestInfo",
+          "description": "the HTTP request info. Usually includes the 'clientRequestId', 'clientIpAddress' (IP address of the user who initiated the event) and 'method' (HTTP method e.g. PUT)."
+        },
+        "level": {
+          "type": "string",
+          "description": "the event level",
+          "enum": [
+            "Critical",
+            "Error",
+            "Warning",
+            "Informational",
+            "Verbose"
+          ],
+          "x-ms-enum": {
+            "name": "EventLevel",
+            "modelAsString": false
+          }
+        },
+        "resourceGroupName": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790546.aspx"
+          },
+          "type": "string",
+          "description": "the resource group name of the impacted resource."
+        },
+        "resourceProviderName": {
+          "$ref": "#/definitions/LocalizableString",
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790572.aspx"
+          },
+          "description": "the resource provider name of the impacted resource."
+        },
+        "resourceId": {
```

Comment:
what is the difference between the property "id" described above and the "resourceId"?